### PR TITLE
Add query parameter to the edge invitations

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

**Comment by melisabok** [comment]
I'm going to set the reviewers group id for now to demo ICLR, but we will need to discuss how to reuse the same invitation between acs and reviewers, or create different invitations.

**Comment by mspector** [comment]
we can talk about it later, but my first reaction is that creating multiple invitations for different groups is ok

File: openreview/conference/matching.py [hunk 1]
```diff
@@ -28,10 +28,16 @@ def _create_edge_invitation(self, id):
                 },
                 'content': {
                     'head': {
-                        'type': 'Note'
+                        'type': 'Note',
+                        "query" : {
```
Comment (zbialecki) [inline openreview/conference/matching.py]: Should be single quotes

Comment (melisabok) [reply openreview/conference/matching.py]: thanks, fixed.

**Pull request merged**